vms/platformvm/state/mock_state.go

```diff
@@ -320,6 +320,21 @@ func (mr *MockStateMockRecorder) GetCurrentValidator(arg0, arg1 interface{}) *go
 	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "GetCurrentValidator", reflect.TypeOf((*MockState)(nil).GetCurrentValidator), arg0, arg1)
 }
 
+// GetDelegateeReward mocks base method.
+func (m *MockState) GetDelegateeReward(arg0 ids.ID, arg1 ids.NodeID) (uint64, error) {
```

Comment:
q: AFAIK the mock_gen.sh script does not update this mock automatically (it does it for mock_chain and mock_diff). Is this change done manually?

Reply:
mockgen doesn't work for files including cgo or generics. For any interfaces using one of those we can use mockgen to get 99% of the way there - but need to fix some final formatting issues.